app/assets/javascripts/discourse/app/components/user-menu/notifications-list.js

```diff
@@ -13,10 +13,19 @@ import Notification from "discourse/models/notification";
 import UserMenuReviewable from "discourse/models/user-menu-reviewable";
 import I18n from "discourse-i18n";
 
+const MAX_LIMIT = 60;
```

Comment:
This is duplicating a value on the server side. If the value changes on the server side this goes out of sync.

Reply:
Yeah I know.. It was bumped 6 months ago by you to 60, but before was 50 for 9 years. I don't think there is much risk here.